docs/instrumentation-list.yaml

```diff
@@ -4176,75 +4176,328 @@ libraries:
       - io.javalin:javalin:[5.0.0,)
   jaxrs:
   - name: jaxrs-1.0
-    disabled_by_default: true
```

Comment:
while technically this instrumentation is disabled by default in the sense that it requires `otel.instrumentation.common.experimental.controller-telemetry.enabled`, I want to use the `disabled_by_default` condition to only apply to instrumentation modules that set the actual module disabled with the standard `enabled` flag. 

I have plans for a different way for us to identify instrumentations that aren't necessarily "disabled by default", but that are otherwise entirely gated by an experimental flag